server/tests/src/test/resources/keycloak/infinispan-rhsso-realm.json

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
This json was exported without secrets. You should use KeyCloak's export tool https://github.com/keycloak/keycloak-documentation/blob/master/server_admin/topics/export-import.adoc